extensions/default/src/DicomWebDataSource/utils/clientManager.ts

```diff
@@ -0,0 +1,249 @@
+import { api } from 'dicomweb-client';
+import StaticWadoClient from './StaticWadoClient';
+import dcm4cheeReject from '../dcm4cheeReject';
+import { errorHandler, utils } from '@ohif/core';
+
+/**
+ * This object plays the central role in OHIF's multiple server handling ability.
+ * It stores all servers configurations and handles all request headers generations
+ */
+interface ConfigToAdd {
+  url: string;
```

Comment:
we didn't have url before. Can you explain what is this?

Reply:
is this `wadoUriRoot`?

Reply:
The dicomweb-client expects a url field

Reply:
where is wadoUriRoot then?